.travis.yml

```diff
@@ -8,14 +8,18 @@ jobs:
         - docker
       before_install:
         - sudo apt install jq
+        - chmod +x ./e2e-tests/stream-mode-tests.sh
```

Comment:
You should be able to avoid this by making the file executable before adding to git.